base_technical_user/README.rst

```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================
+Base Technical User
+===================
+
+This module extends the functionality of company management.
+It allows you to bind a technical user on the company in order to use it in
+batch processes.
+
+The technical user must
+- be inactive to avoid login
```

Comment:
If highly advise not to do that, been there, done that.

* you set some groups on your technical user, let's say one of the group is 'connector manager', all works well
* someday, you open the groups menu, edit the 'connector manager' group and add a user in the many2many, save it
* your technical user is no longer part of the 'connector manager' group and you don't notice it until things start to fail weirdly 

What happened? the many2many for users on the group do not include inactive users, so you just removed all the inactive users from the group by adding a new one.

What we do now is that we don't set a password on the user, it can't be used to log in.